elements/all/test/index.html

```diff
@@ -9,8 +9,11 @@
     <script>
       // Load and run all tests (.html, .js):
       WCT.loadSuites([
-        'fouc-test.html'
+        'fouc-test.html',
+        // 'module-amd-test.html',
```

Comment:
@mwcz - Do we want to leave these commented out?

Reply:
Dang, no, they shouldn't be.  Those commented test files aren't written yet though, I just stubbed out the filenames.  I should finish those before merging this PR.